Kotlin/lib/src/main/java/com/wolt/blurhashkt/BlurHashDecoder.kt

```
@@ -32,14 +30,14 @@ object BlurHashDecoder {
         val numY = (sizeFlag / 9) + 1
         val numX = (sizeFlag % 9) + 1
 
-        val quantisedMaximumValue = blurHash[1].toString().decode83()
-        val maximumValue = (quantisedMaximumValue + 1) / 166f
-
         if (blurHash.length != 4 + 2 * numX * numY) {
             return null
         }
```

fail early - move the computations that are not needed to detect this case down